### DISCUSSION TITLE
How to properly test this component with Jest and React Testing Library?

### DISCUSSION THREAD

**Discussion by marcosclorian** [wojtekmaj/react-date-picker#633] [Q&A]
Hi! I'm trying to test this component by doing this: 

`const inputsDate = await screen.findAllByRole("spinbutton");`

In this way, I can see every input number and select one by one, there is another way to test it?

**Comment by wojtekmaj** [comment]
I would rather try and find it using accessible label, and that's the way RTL usually recommends finding elements.
